_sass/notion/_alerts.scss

```diff
@@ -1,6 +1,6 @@
 .alert.alert-warn {
     background-color: rgba(255,229,100,.3);
-    border-color: #e7c000;
+    color-primary-very-light: #e7c000;
```

Comment:
I don't understand this line... isn't `color-primary-very-light` a SASS variable, not a CSS key?

Reply:
Umm, why, yes. That’ll teach me to use regexes and not double check what they did. (Also: CSS is tooooo forgiving.)